scripts/dataset/export/index.js

```diff
@@ -0,0 +1,124 @@
+import fsApi from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import archiver from 'archiver';
+import config from 'config';
+
+import GitAdapter from '../../../src/storage-adapters/git/index.js';
+import MongoAdapter from '../../../src/storage-adapters/mongo/index.js';
+import * as renamer from '../../utils/renamer/index.js';
+import readme from '../assets/README.template.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const locale = 'en-EN';
+const dateOptions = { year: 'numeric', month: 'long', day: 'numeric' };
+
+export default async function generateArchive({ archivePath, releaseDate }) {
+  const versionsStorageAdapter = await initVersionsStorageAdapter();
+  const { archive, archiveName, archiveWrittenPromise } = await initializeArchive(archivePath);
+
+  await renamer.loadRules();
+
+  const periodDates = { first: null, last: null };
+  const services = new Set();
+
+  for await (const record of versionsStorageAdapter.iterate()) {
+    const { serviceId, documentType, ...othersRecordAttributes } = record;
+    const { serviceId: renamedServiceId, documentType: renamedDocumentType } = renamer.applyRules(serviceId, documentType);
+
+    addRecordToArchive({
+      record: { serviceId: renamedServiceId, documentType: renamedDocumentType, ...othersRecordAttributes },
+      periodDates,
+      services,
+      archive,
+      archiveName,
+    });
+  }
+
+  addReadmeAndLicenseToArchive({ releaseDate, periodDates, services, archive, archiveName });
```

Comment:
```suggestion
  addAssets({ releaseDate, periodDates, services, archive, archiveName });
```

…but honestly, I am not convinced of the added value of this function. I'd recommend to either put all the body in this function, or split into `addReadme` and `addLicense`.